component/main.jsonnet

```diff
@@ -45,7 +46,8 @@ local deployment = kube.Deployment('eventrouter') {
         containers_+: {
           'kube-eventrouter': kube.Container('kube-eventrouter') {
             image: params.images.eventrouter.registry + '/' + params.images.eventrouter.image + ':' + params.images.eventrouter.tag,
-            imagePullPolicy: 'Always',
+            imagePullPolicy: 'IfNotPresent',
```

Comment:
I would probably remove the imagePullPolicy completely and let Kubernetes select it based on the image tag, but no hard preference either way.